vividus-plugin-rest-api/src/main/java/org/vividus/http/variable/ResponseAsBytesDynamicVariable.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.http.variable;
+
+import org.vividus.http.HttpTestContext;
+import org.vividus.http.client.HttpResponse;
+
+public class ResponseAsBytesDynamicVariable extends AbstractHttpResponseDynamicVariable
+{
+    public ResponseAsBytesDynamicVariable(HttpTestContext httpTestContext)
```

Comment:
why is one more variable needed? can't we use `DataWrapper` to keep response data?